src/tunnels/views.py

```diff
@@ -12,6 +13,30 @@
 
 from authorized_keys.models import ReverseServerAuthorizedKeys
 
+from authorized_keys.utils import ssh
+from tunnel_script_renderer import ssh_tunnel_script_factory
+from tunnel_script_renderer import sshd_client_config_factory, sshd_server_config_factory
+
+## Static constants
+CLIENT_STEM = """
+# ========================================
+# Endpoint Configuration
```

Comment:
I am uncertain about this naming. 🤔
I do not believe that `Endpoint` is appropriate in my original code.
Would replacing it with `Tunnel` be more suitable?

Reply:
It's OK : D